cypress/support/helpers.ts

```diff
@@ -1,59 +1,26 @@
-import {
-  countries,
-  dataset,
-  datasetOverview,
-  harvestData,
-  languages,
-  records,
-  report,
-  runningExecutions,
-  statistics,
-  user,
-  workflow,
-  workflowExecutions,
-  xslt
-} from '../fixtures';
+import { User } from '../../src/app/_models/user';
 
 export function setupUser(): void {
   cy.window().then((w) => {
-    w.localStorage.setItem(
-      'currentUser',
-      JSON.stringify({
-        user,
-        email: user.email,
-        token: user.metisUserAccessToken.accessToken
-      })
+    cy.request('POST', Cypress.env('dataServer') + '/users/data/authenticate.json').then(
+      (response) => {
+        const user = response.body as User;
+        w.localStorage.setItem(
```

Comment:
It's been that way since the beginning, but I've added a cleanupUser function with an "afterEach" hook (note: "afterAll" wasn't found, but it's set up in a beforeEach so it's fine to remove it there)